stix_shifter_utils/stix_transmission/utils/RestApiClient.py

```diff
@@ -103,8 +103,7 @@ def call_api(self, endpoint, method, headers=None, data=None, urldata=None, time
                 url = 'https://' + self.server_ip + '/' + endpoint
             try:
                 session = requests.Session()
-                retry_strategy = Retry(total=self.retry_max, backoff_factor=0, status_forcelist=[429, 500, 502, 503, 504],
-                                       allowed_methods=["HEAD", "GET", "PUT", "DELETE", "OPTIONS", "TRACE"])
+                retry_strategy = Retry(total=self.retry_max, backoff_factor=0, status_forcelist=[429, 500, 502, 503, 504])
```

Comment:
what is the error when allowed_methods is specified? Is there any specific method that infoblox need?

Reply:
what version of `urllib3` and `requests` are you using? seems like you may need to upgrade these specified packages in your python env.

Reply:
I'll close the PR.
Thanks again